FreeRTOS/FreeRTOS-Plus-TCP
# Added Network Interface driver for TI TM4C1294NCPDT and Compiler supp…

…ort for TI Code Composer Studio.

Network Interface driver for TI TM4C1294NCPDT

Description
-----------
Added NIC driver for TM4C1294NCPDT to FreeRTOS+TCP stack.

Test Steps
-----------
Not an issue, but a contribution to the project. To test the implementation, acquire the TI EK-TM4C1294XL evaluation kit, compile FreeRTOS and FreeRTOS+TCP and include the TCP Echo Server module.

Related Issue
-----------
N/A


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
